acceptance/tests/direct_puppet/catalog_uuid_correlates_catalogs_with_reports.rb

```diff
@@ -1,4 +1,9 @@
 test_name "PUP-5872: catalog_uuid correlates catalogs with reports" do
+
+  tag 'audit:medium',
+      'audit:acceptance',
+      'server'
```

Comment:
doesn't this just test that the uuid is the same in catalog and report when running the agent with --use-cached-catalog?

great abstractions in this test make it very easy to read!